### PR TITLE
cmd/snap: install *.snap and *.snap.* as files too

### PR DESCRIPTION
**Pull request by niemeyer**

### CODE REVIEW

**Comment by niemeyer** [comment]
retest this please

File: cmd/snap/cmd_snap_op.go [hunk 1]
```diff
@@ -118,10 +118,11 @@ func (x *cmdInstall) Execute([]string) error {
 	var err error
 
 	cli := Client()
-	if strings.Contains(x.Positional.Snap, "/") {
-		uuid, err = cli.InstallSnapFile(x.Positional.Snap)
+	name := x.Positional.Snap
+	if strings.Contains(name, "/") || strings.HasSuffix(name, ".snap") || strings.Contains(name, ".snap.") {
```
Comment (zyga) [inline cmd/snap/cmd_snap_op.go]: What is the third one for? (``*.snap.*``)

**Comment by zyga** [comment]
+1, just one question

File: cmd/snap/cmd_snap_op.go (hunk 1 above)
Comment (niemeyer) [inline cmd/snap/cmd_snap_op.go]: Common ending for repeated browser downloads and the such (foo.snap.1, etc).  Cannot be ambiguous with a real snap name, due to the two dots.

**Comment by mvo5** [comment]
retest this please

**Comment by mvo5** [comment]
:+1:

**Comment by mvo5** [comment]
retest this please

File: cmd/snap/cmd_snap_op.go (hunk 1 above)
Comment (chipaca) [inline cmd/snap/cmd_snap_op.go]: @niemeyer note chrome adds ` (n)` to the filename, e.g. `http.chipaca_4.66920_all (1).snap`

**Pull request merged**